fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/predicates/RangeConstraints.java

```diff
@@ -533,6 +533,15 @@ public Range<Boundary> compile(@Nonnull final EvaluationContext evaluationContex
             return range;
         }
 
+        @Override
+        public int planHash(@Nonnull final PlanHashMode mode) {
+            int hashCode = 0;
+            for (final var compilableComparison : compilableComparisons) {
+                hashCode += 31 * compilableComparison.planHash(mode);
+            }
+            return hashCode;
+        }
```

Comment:
This is not order-sensitive if I am reading this right.

Reply:
Oh, I see; I was confused, because usually a hash like this would be:

```java
hashCode = 31 * hashCode + compilableComparison.planHash(mode);
```

And I'd auto-completed that in my mind. I think this is fine, though it is still weird to me that this isn't using `PlanHashable.setsPlanHash`